coops-ui/tests/unit/Directors.spec.ts

```diff
@@ -191,39 +198,39 @@ describe('Directors as a COOP', () => {
     await vm.editDirectorName(0)
 
     // Verify the correct data in the text input field
-    expect(vm.$el.querySelectorAll('.edit-director__first-name input')[0].value).toBe('Joe')
+    expect(vm.$el.querySelectorAll('.edit-director__first-name input')[0].value).toBe('Peter')
 
     // Change the text input
     setValue(vm, '.edit-director__first-name input', 'Steve')
 
     // Click and save the updated data
-    await vm.saveEditDirector(1, 2)
+    await vm.saveEditDirector(0, 1)
 
     // Verify the updated text field value
     expect(vm.$el.querySelectorAll('.edit-director__first-name input')[0].value).toBe('Steve')
 
     // Verify that only Change Name component is shown
-    expect(vm.editFormShowHide.showAddress).toEqual(false)
+    expect(vm.editFormShowHide.showAddress).toEqual(true)
     expect(vm.editFormShowHide.showName).toEqual(true)
-    expect(vm.editFormShowHide.showDates).toEqual(false)
+    expect(vm.editFormShowHide.showDates).toEqual(true)
```

Comment:
So now all 3 components are showing? That does not seem correct. At a minimum, the comment on line 212 should be updated, but I don't understand why this test would be so fundamentally different from before?

(If these fixes are not due to your code then I recommend you don't make these changes and instead create a ticket to fix these tests.)